### Available Datasets 

**TCGA** 
  + [GBM](https://tcga-data.nci.nih.gov/tcga/tcgaCancerDetails.jsp?diseaseType=GBM&diseaseName=Glioblastoma%20multiforme) ^1
  + [LGG](https://tcga-data.nci.nih.gov/tcga/tcgaCancerDetails.jsp?diseaseType=LGG&diseaseName=Brain%20Lower%20Grade%20Glioma) ^2 
  + [cBioPortal](http://www.cbioportal.org/public-portal/index.do) ^3 
  
**Rembrandt**
  + [caArray](https://array.nci.nih.gov/caarray/project/fine-00037)
  + [Repository for Molecular Brain Neoplasia Data](https://caintegrator.nci.nih.gov/rembrandt/) ^4

**Gravendeel**
  + [GSE16011](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE16011) ^5

**Phillips**
  + [GSE4271](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE4271) ^6

**Freije**
  + [GSE4412](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE4412) ^7
  
**Murat**
  + [GSE7696](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE7696) ^8  

**Reifenberger**
  + [GSE53733](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE53733) ^9  
  
**Bao**
  + [GSE48865](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48865) ^10    
  
**Gill**
  + [GSE59612](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59612) ^11 

**Gorovets**
  + [GSE35158](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35158) ^12 

**Nutt**
  + [Broad](http://www.broadinstitute.org/cgi-bin/cancer/datasets.cgi) ^13  

**Ducray**
  + [E-TABM-898](http://www.ebi.ac.uk/arrayexpress/experiments/E-TABM-898) ^14 
  

References:  
1. [The Cancer Genome Atlas Research Network 2008](http://www.ncbi.nlm.nih.gov/pubmed/18772890)  
2. unpublished, see [TCGA guidelines](http://cancergenome.nih.gov/publications/publicationguidelines) for unpublished data  
3. [Cerami E. et al. 2012](http://www.ncbi.nlm.nih.gov/pubmed/22588877)  
4. [Madhavan S. et al. 2009](http://www.ncbi.nlm.nih.gov/pubmed/19208739)  
5. [Gravendeel LA. et al. 2009](http://www.ncbi.nlm.nih.gov/pubmed/19920198)  
6. [Phillips HS. et al. 2006](http://www.ncbi.nlm.nih.gov/pubmed/16530701)  
7. [Freije WA. et al. 2004](http://www.ncbi.nlm.nih.gov/pubmed/15374961)  
8. [Murat A. et al. 2008](http://www.ncbi.nlm.nih.gov/pubmed/18565887)  
9. [Reifenberger G. et al. 2014](http://www.ncbi.nlm.nih.gov/pubmed/24615357)  
10. [Bao ZS. et al. 2014](http://www.ncbi.nlm.nih.gov/pubmed/25135958)  
11. [Gill BJ. et al. 2014](http://www.ncbi.nlm.nih.gov/pubmed/25114226)  
12. [Gorovets D. et al. 2012](http://www.ncbi.nlm.nih.gov/pubmed/22415316)  
13. [Nutt CL. et al. 2003](http://www.ncbi.nlm.nih.gov/pubmed/12670911)    
14. [Ducray F. et al. 2010](http://www.ncbi.nlm.nih.gov/pubmed/20822523)  


### Data processing  
**mRNA analysis**       
* Affymetrix expression array analyses were completed in R using the [Bioconductor](http://bioconductor.org) suite. The [‘affy’](http://www.bioconductor.org/packages/release/bioc/manuals/affy/man/affy.pdf) package was used for robust multi-array average normalization followed by quantile normalization.   TO ADD RNA-SEQ   

**Copy number**     
* Copy number alterations (obtained through the [GISTIC 2.0](http://www.broadinstitute.org/cgi-bin/cancer/publications/pub_paper.cgi?mode=view&paper_id=216&p=t) algorithm) were downloaded from the [cBioPortal](http://www.cbioportal.org/public-portal/index.do) using the[‘cgdsr’](http://cran.r-project.org/web/packages/cgdsr/cgdsr.pdf) package.   

**Survival**         
* Kaplan–Meier survival analysis were completed using the [‘survival’](http://cran.r-project.org/web/packages/survival/survival.pdf) package in R. Hazard ratios were determined utilizing the ‘coxph’ function from the ‘survival’ package. To generate HR plot we partially use some code previously describend in [*Cutoff finder*](http://molpath.charite.de/cutoff/index.jsp).    

**Classification**    
* The Subtype call generated applying a **Support Vector Machine (SVM)** algorithm, is done performing a linear Kernel with 10-fold cross validation, using the ‘ksvm’ function of the [‘kernlab’](http://cran.r-project.org/web/packages/kernlab/kernlab.pdf) package. For GBM samples classification we use 5 subtypes (Classical, Mesenchymal, Neural, Proneural, G-CIMP) as defined in [Brennan C. et al. 2013](http://www.ncbi.nlm.nih.gov/pubmed/24120142).
* The **K-nearest neighbor (K-NN)** Subtype calls, are obtained using the ‘knn’ function of the [‘class’](http://cran.r-project.org/web/packages/class/class.pdf) package. For GBM samples classification we use the 5 subtypes as in the SVM calls.  
* The Subtype call generated  by single sample **Gene Set Enrichment Analysis (ssGSEA)**, is done using the ‘gsva’ function of the [‘gsva’](http://www.bioconductor.org/packages/2.12/bioc/html/GSVA.html) package. For GBM samples classification we uses 4 subtypes (Classical, Mesenchymal, Neural, Proneural) and a 840 gene classifier as defined in [Verhaak RG. et al. 2010](http://www.ncbi.nlm.nih.gov/pubmed/20129251).

### App info

Version 0.10 (10/01/2014)

By: [Massimo Squatrito](http://www.cnio.es/ing/grupos/plantillas/presentacion.asp?grupo=50008938) and [Robert Bowman](http://www.mskcc.org/research/lab/johanna-joyce/member/robert-bowman)

Built using [Shiny](http://www.rstudio.com/shiny) by [RStudio](http://www.rstudio.com)

Code available on [GitHub](https://#github.com/msquatrito/shiny_GlioVis)

### License

The GlioVis is licensed under the [AGPLv3](http://www.tldrlegal.com/l/AGPL3) 

As a summary, the AGPLv3 license requires, attribution, include copyright and license in copies of the software, state changes if you modify the code, and disclose all source code. 

&copy; Massimo Squatrito (2014) 
<a rel="license" href="http://www.creativecommons.org/licenses/by-nc/4.0/"><img alt="Creative Commons License" style="border-width:0" src="cc.png" /></a><br />This work is licensed under a <a rel="license" href="http://www.creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International License</a>. 
  
### Funding

<a href ="http://www.seveballesteros.com"><img src="seve.jpg" height = "135"  width = "405"></a>

